core/src/Feature.h

```diff
@@ -45,6 +45,8 @@ class VEYON_CORE_EXPORT Feature
 		Session = 0x0004,
 		Internal = 0x0008,
 		Option = 0x0010,
+        NoContext = 0x0012,
+        NoMenu = 0x0014,
```

Comment:
Flags always should have one bit set only so they can be ORed. So it goes like 0x0010, 0x0020, 0x0040, 0x0080 etc. -> 0x0020 is already used by the `Checked` flag. Also take care of the whitespaces for indentation ;-)

Reply:
Sorry for the whitespace trouble again. I'll try to find the Qt creator setting now instead of trying to fix it manually and obviously fail here and even not noticing it before sending the PR.

Reply:
Should I still supply this change when it is corrected, so it can be used in the future if desired?